pints/_mcmc/_global_adaptive_covariance.py

```diff
@@ -0,0 +1,68 @@
+#
+# Base class for Adaptive covariance MCMC methods
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2018, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+
+
+class GlobalAdaptiveCovarianceMCMC(pints.AdaptiveCovarianceMCMC):
+    """
+    Base class for single chain MCMC methods that globally adapt a
+    proposal covariance matrix when running, in order to control the
+    acceptance rate.
+
+    *Extends:* :class:`AdaptiveCovarianceMCMC`
+    """
+    def __init__(self, x0, sigma0=None):
+        super(GlobalAdaptiveCovarianceMCMC, self).__init__(x0, sigma0)
+
+    def ask(self):
+        """ See :meth:`SingleChainMCMC.ask()`. """
+        super(GlobalAdaptiveCovarianceMCMC, self).ask()
```

Comment:
That can't be right :D

Reply:
There doesn't seem to be anything in that method?

Reply:
There is an initialise call in the method above that method (it also calls super, to reach AdaptiveCovarianceMCMC.ask()).